src/lib.rs

```diff
@@ -1036,16 +1036,19 @@ impl Instance {
         }
     }
 
+    /// Creates a surface from `CoreAnimationLayer`.
+    ///
+    /// # Safety
+    ///
+    /// - layer must be a valid object to create a surface upon.
     #[cfg(any(target_os = "ios", target_os = "macos"))]
     pub unsafe fn create_surface_from_core_animation_layer(
         &self,
         layer: *mut std::ffi::c_void,
     ) -> Surface {
         let surface = wgc::instance::Surface {
             #[cfg(feature = "vulkan-portability")]
-            vulkan: self.context.instance.vulkan.as_ref().map(|inst| {
-                inst.create_surface_from_layer(layer as *mut _, cfg!(debug_assertions))
-            }),
+            vulkan: None, //TODO: create_surface_from_layer ?
```

Comment:
Is this not used in most applications?

Reply:
This was not compiling, since Vulkan backend doesn't currently have `create_surface_from_layer` at all. We can add it if requested, but for now leaving that dead code was hurting people who would just do `create_surface`